src/core/style/mixins.scss

```diff
@@ -326,3 +326,47 @@
     }
   }
 }
+
+// Only use when in row layout
+@mixin when-layout-row($element) {
```

Comment:
Right, I do have some PRs that would use this to fix issues with md-select, md-checkbox, md-input-container and md-datepicker. I had a tough time trying to squish it down though.